src/node/net.ts

```diff
@@ -948,13 +950,9 @@ function initializeConnection(
   let { port = 0 } = options;
 
   if (autoSelectFamily != null) {
-    // We don't support this option. If the value is falsy, we can safely ignore it.
-    // If the value is truthy, we'll throw an error.
-    throw new ERR_INVALID_ARG_VALUE(
-      'options.autoSelectFamily',
-      autoSelectFamily,
-      'is not supported'
-    );
+    // We don't support this option.
+    // We shouldn't throw this because services like mongodb depends on it.
+    // TODO(soon): Investigate supporting this.
```

Comment:
Just noting... I'm not really sure it's really possible to support this as the runtime has no visibility into how the backend connection is actually established.